Experiment_Launcher_code/ExperimentLauncher.py

```diff
@@ -11,25 +11,26 @@ def main():
 
     if settings:
         # Extract the necessary parameters from the settings
+        experiment_name = settings.get('experiment_name')
```

Comment:
is this and the comport an  input from the user?

Reply:
Yes. we discussed COM port above. The reason I added the experiment name is to be able to create an experiment specific log file (already made this change in the logger on my local, it waits to be merged :-). This way you do not need to change the file name ion the code every time you run an experiment, or remember to move the file if you want to keep it, without changing the names.